roadmap.md

```diff
@@ -0,0 +1,87 @@
+# Nix is everywhere and invisible
+
+Software developers use Nix as a matter of course every day, mostly without even noticing.
+Nix runs trivially, anywhere.
+
+For individuals to large organizations, Nix underpins the the entire software supply chain:
+- Developer tooling
+- Build automation
+- Binary distribution.
+
+To this end, the Nix team will work towards the following goals.
+
+## Make Nix easy to adopt
+
+  - Well-defined target user base
```

Comment:
Not sure what this means...

Reply:
Wo is the target audience? What are these users' needs? This may entail sketching sobering like user personas.

Reply:
(Eelco) should that be defined before we can have a clear vision? So that we can know where we want to lead Nix